packages/terra-toolkit/src/scripts/README.md

```diff
@@ -0,0 +1,15 @@
+#nightwatch
```

Comment:
I will move this documentation into the README in the docs directory.  As well as add information about update-sauce.js